DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/README.md

```diff
@@ -207,8 +207,8 @@ Example output when running on Intel® PAC with Intel Arria® 10 GX FPGA for 327
 Device name: pac_a10 : Intel PAC Platform (pac_f000000)
 Generating 32768 random matrices
 Running QR decomposition of 32768 matrices repeatedly
-   Total duration:   43.3474 s
-Throughput: 24.1901k matrices/s
+   Total duration:   42.3408 s
+Throughput: 24.7652k matrices/s
```

Comment:
I don't think there's a need to update this with every seed change.  This is just 'sample output', I don't think it has to match exactly.

Reply:
I won't change it again :) The previous throughput was below the minimum requirement so I should probably update it just this once.